src/quo2/reanimated.cljs

```diff
@@ -67,3 +78,9 @@
 (defn animate-shared-value-with-delay [anim val duration easing delay]
   (set-shared-value anim (with-delay delay (with-timing val (js-obj "duration" duration
                                                                     "easing"   (get easings easing))))))
+
+(defn interpolate [anim-value input-range output-range options]
+  (let [input-range (clj->js input-range)
+        output-range (clj->js output-range)
+        options (clj->js (map-keys kebab-case->camelCase options))]
+    (.interpolate ^js reanimated (oget anim-value "value") input-range output-range options)))
```

Comment:
Is this still required?

Reply:
No, but shouldn't we keep it, in case somebody wants to use the `interpolate` function from reanimated in a "Clojure way"?